packages/diffhtml-render-to-string/index.js

```diff
@@ -1,4 +1,70 @@
-function renderAttributes(attributes) {
+const { use, Internals, createTree, innerHTML } = require('diffhtml');
+
+// Simple helper to make a DOM-like object.
+const makeDomNode = (props) => ({
+  ...props,
+
+  childNodes: [],
+  parentNode: {},
+
+  appendChild(node) {
+    this.childNodes.push(node);
+  }
+});
+
+// Apply a transformation function against the Nodes to produce a string of
+// markup.
+use({
+  createNodeHook(vTree) {
+    return makeDomNode(vTree);
+  },
+});
+
+const tasks = new Set(Internals.defaultTasks);
+
+// Remove incompatible tasks, and replace them with serialization-compatible
```

Comment:
While this works, I think it would be smarter to simply replace the `reconcileTrees` task and remove the `patchNode` task. There is too much shuffling and required knowledge about existing tasks for this to work in all cases.